.docker/test.ini

```diff
@@ -97,7 +97,7 @@ ckan.redis.url = redis://redis:6379
 #       Add ``datapusher`` to enable DataPusher
 #       Add ``resource_proxy`` to enable resource proxying and get around the
 #       same origin policy
-ckan.plugins = stats text_view image_view recline_view publications_qld_theme qgovext resource_type_validation
+ckan.plugins = stats text_view image_view recline_view publications_qld_theme qgovext csrf_filter resource_type_validation ssm_config
```

Comment:
I think the ssm config may be slowing it down

Reply:
Eh, there have been 15-minute builds before.

Reply:
the ssm logging should be turned off by default for us in both tests and on our box. (i'd also set the config options to disable it in testing)